src/app/components/elements/Dashboard.tsx

```diff
@@ -45,28 +46,60 @@ export const Dashboard = ({
     );
   };
 
-  const ShowMeButtons = ({ className }: { className?: string }) => (
-    <Container id="homepageButtons" className={`${className} ${!user?.loggedIn ? "pt-0 px-lg-0" : ""}`}>
-      <h3>Show me</h3>
-      <Row>
-        <Col xs={12} lg={user?.loggedIn ? 12 : 4} className="py-1">
-          <Button size="lg" tag={Link} to={"/topics/gcse"} color="secondary" block>
-            GCSE resources
-          </Button>
-        </Col>
-        <Col xs={12} lg={user?.loggedIn ? 12 : 4} className="py-1">
-          <Button size="lg" tag={Link} to={"/topics/a_level"} color="secondary" block>
-            A Level resources
-          </Button>
-        </Col>
-        <Col xs={12} lg={user?.loggedIn ? 12 : 4} className="py-1">
-          <Button size="lg" tag={Link} to={"/events"} color="secondary" block>
-            Events
-          </Button>
-        </Col>
-      </Row>
-    </Container>
-  );
+  const ShowMeButtons = ({ className }: { className?: string }) => {
```

Comment:
You should probably look into refactoring this component, because this is no longer `ShowMeButtons`, this is show me buttons plus a lot of extra functionality.

Reply:
I broke this up now into `DashboardButtons`, then `ShowMeButtons` and `TeacherButtons`